components/advocates/AdvocateCard.vue

```diff
@@ -37,6 +37,7 @@ export default class extends Vue {
   @Prop(String) country!: any
   @Prop(String) region!: any
   @Prop(String) slackId!: any
+  @Prop(String) slackUsername!: any
```

Comment:
If it's known that the Slack username is a string, we should probably also set the TS type accordingly:

```suggestion
  @Prop(String) slackUsername!: string
```